docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit/add-certificate-self.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Add self-signed certificates to YugabyteDB Anywhere
+headerTitle: Add certificates
+linkTitle: Add certificates
+description: Add self-signed certificates to YugabyteDB Anywhere.
+headcontent: Use your own certificates for encryption in transit
+menu:
+  preview_yugabyte-platform:
+    parent: enable-encryption-in-transit
+    identifier: add-certificate-1-self
+    weight: 20
+type: docs
+---
+
+{{<tabs>}}
+{{<tabitem href="../add-certificate-self/" text="Self-Signed" active="true" >}}
+{{<tabitem href="../add-certificate-ca/" text="CA-Signed" >}}
+{{<tabitem href="../add-certificate-hashicorp/" text="Hashicorp Vault" >}}
+{{<tabitem href="../add-certificate-kubernetes/" text="Kubernetes" >}}
+{{</tabs>}}
+
+Instead of using YugabyteDB Anywhere-provided certificates, you can use your own self-signed certificates that you upload to YugabyteDB Anywhere.
+
+## Prerequisites
+
+The certificates must meet the following criteria:
+
+- Be in the `.crt` format and the private key must be in the `.pem` format, with both of these artifacts available for upload.
+- Contain IP addresses of the target database nodes or DNS names as the Subject Alternative Names (wildcards are acceptable).
+
+YugabyteDB Anywhere produces the node (leaf) certificates from the uploaded certificates and copies the certificate chain, leaf certificate, and private key to the nodes in the cluster.
+
+### Convert certificates and keys from PKCS12 to PEM format
+
+If your certificates and keys are stored in the PKCS12 format, you can convert them to the PEM format using OpenSSL.
+
+Start by extracting the certificate via the following command:
+
+```sh
+openssl pkcs12 -in cert-archive.pfx -out cert.pem -clcerts -nokeys
+```
+
+To extract the key and write it to the PEM file unencrypted, execute the following command:
+
+```sh
+openssl pkcs12 -in cert-archive.pfx -out key.pem -nocerts -nodes
+```
+
+If the key is protected by a passphrase in the PKCS12 archive, you are prompted for the passphrase.
+
+### Verify certificate chain
```

Comment:
This whole section needs to move to the CA-signed certs page, which is where they are responsible for the node certs. This section tells them how they can verify their node certs are indeed accurate.